tests/rustdoc/extremely_long_typename.extremely_long_typename.html

```diff
@@ -0,0 +1 @@
+<li><div class="item-name"><a class="struct" href="struct.CreateSubscriptionPaymentSettingsPaymentMethodOptionsCustomerBalanceBankTransferEuBankTransfer.html" title="struct extremely_long_typename::CreateSubscriptionPaymentSettingsPaymentMethodOptionsCustomerBalanceBankTransferEuBankTransfer">Create<wbr />Subscription<wbr />Payment<wbr />Settings<wbr />Payment<wbr />Method<wbr />Options<wbr />Customer<wbr />Balance<wbr />Bank<wbr />Transfer<wbr />EuBank<wbr />Transfer</a></div></li>
```

Comment:
I'm surprised that it generates `<wbr />` whereas we generate `<wbr>` in rustdoc. Dark magic.

Reply:
Don't worry, I'm not asking to change it or anything. I was just amused to see this change and wondered why this tag was updated this way. But thanks for confirming my suspicions. ;)